secure_message/application.py

```diff
@@ -57,11 +56,11 @@ def create_app(config=None):
     api.add_resource(Info, '/info')
 
     api.add_resource(MessageList, '/messages', '/v2/messages')
-    api.add_resource(MessageSend, '/message/send')
+    api.add_resource(MessageSend, '/message/send', '/v2/messages')
     api.add_resource(MessageById, '/message/<message_id>', '/v2/messages/<message_id>')
```

Comment:
MessageList and MessageById should be removed as they are no longer used. There are some other bits of code that can then be removed e.g. secure_message.authorization.authorizer will no longer be needed.

Reply:
They are technically being used by ras-frontstage-api, but https://github.com/ONSdigital/ras-frontstage-api/pull/64 removes those endpoints because they're not being hit anymore.  I'll remove these ones at the same time as that PR goes in.